File: 05/exercise1-objc/driver.m
#import "car.h"
#import "racecar.h"

int main(int argc, char* argv[]) {
	Car *chevy = [[Car alloc] Name: @"Chevrolette" MaxSpeed: 95 Color: @"black" AmountOfEngineValve: 4];
	Racecar *f1 = [[Racecar alloc] Name: @"Ferrari" Sponsor: @"Bug2Bug" MaxSpeed: 220 Color: @"red" AmountOfEngineValve: 40 AmountOfGearbox: 7 presenceOfParachute: YES];
	[chevy printDetail];
	[f1 printDetail];
	return 0;
}

File: 05/exercise1/driver.cpp
#include "car.h"
#include "racecar.h"

using namespace std;

int main(int argc, char* argv[]) {
	Car *chevy = new Car("Chevrolette", 95, "black", 4);
	Car *f1 = new Racecar("Ferrari", 220, "red", 40, "Bug2Bug", 7, true);
	chevy->printDetail();
	f1->printDetail();
	delete chevy;
	delete f1;
	return 0;
}